core/evancore.vue.libs/src/interfaces.ts

```diff
@@ -17,7 +17,7 @@
   the following URL: https://evan.network/license/
 */
 
-/******************************************** interfaces ******************************************/
+/** ****************************************** interfaces ***************************************** */
```

Comment:
```suggestion

```
Maybe instead remove this convoluted comment all together? I mean we are in the interfaces.ts after all ^^